doc/source/cluster/cloud.rst

```diff
@@ -184,11 +184,55 @@ Private Cluster (List of nodes)
 
 The most preferable way to run a Ray cluster on a private cluster of hosts is via the Ray Cluster Launcher.
 
+.. tip:: To avoid getting the password prompt when running private clusters make sure to setup your ssh keys on the private cluster as follows:
+
+    .. code-block:: bash
+
+        $ ssh-keygen
+        $ cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+
+Manually Managed
```

Comment:
Please provide definitions for what "manually managed" means for the user? Also do this for "automatically managed". For example,

> There are two ways of running Private Clusters:
> - Manually managed, where as a user you will ....
> - Automatically managed, where the cluster launcher will do .... on behalf of the user.